tasks/main.yml

```diff
@@ -19,16 +19,30 @@
 
 - name: Add Phusion apt repo.
   apt_repository:
-    repo: 'deb https://oss-binaries.phusionpassenger.com/apt/passenger {{ansible_distribution_release}} main'
+    repo: "{{ passenger_repo_url }}"
     state: present
     update_cache: yes
 
 # Nginx and passenger installation.
 - name: Install Nginx and Passenger.
   apt: "pkg={{ item }} state=installed"
   with_items:
```

Comment:
For these two tasks, we can still keep one and just have the list of passenger packages be a variable, like:

```yaml
passenger_packages:
  - nginx-extras
  - passenger
```

That way there's only one new task (for the license key) and you can override the `passenger_repo_url`, `passenger_enterprise_license`, and `passenger_packages` to use the enterprise edition.

Could you also add a subsection to the README with instructions for overriding the three variables to use the enterprise edition?

Thanks!